### ISSUE TITLE
Bug: CoordinatePart.TryParse ignores S hemisphere.

### ISSUE THREAD

**Issue by FanciestW** [Tronald/CoordinateSharp#62]
The `TryParse` function withing the `CoordinatePart` class does not correctly parse a value that is in degree decimal minutes format but only has a hemisphere and a degree value. So for example: "S 37". This input will always parse the hemisphere and change it to N. So "S 37" will be outputted as "N 37 0".
`CoordinatePart.TryParse("S 37", CoordinateType.Lat, out var coordPart);`
Above the value of coordPart will have a `toDouble()` of 37 but should really be -37.

**Comment by Tronald** [comment]
@FanciestW Yep! Somewhere along the line I guess I forgot indexes start at 0 like some sort of Jabroni!

The bug is only effecting DD formats. Signed, DDM and DMS don't present this bug.

It's fixed in 1.1.3.8. Here is a [Compiled Version] of the library if you want to swap it out for now.(https://github.com/Tronald/CoordinateSharp/blob/v1.1.3.8/CoordinateSharp%20v1_1_3_8%20(ALPHA).zip).

Thank you for bringing this to my attention. Leaving open until I release 1.1.3.8.
